src/components/plots/barPlot.ts

```diff
@@ -179,12 +184,21 @@ export module Plot {
           var secondaryDist = 0;
 
           // if we're inside a bar, distance in both directions should stay 0
-          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox)) {
+          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox, tolerance)) {
             var plotPtPrimary = this._isVertical ? plotPt.x : plotPt.y;
-            var plotPtSecondary = this._isVertical ? plotPt.y : plotPt.x;
-
             primaryDist = Math.abs(queryPtPrimary - plotPtPrimary);
-            secondaryDist = Math.abs(queryPtSecondary - plotPtSecondary);
+
+            // compute this bar's min and max along the secondary axis
+            var barMinSecondary = this._isVertical ? barBBox.y : barBBox.x;
+            var barMaxSecondary = barMinSecondary + (this._isVertical ? barBBox.height : barBBox.width);
+
+            if (queryPtSecondary >= barMinSecondary - tolerance && queryPtSecondary <= barMaxSecondary + tolerance) {
```

Comment:
This makes sense from a code standpoint, but from a semantical standpoint, can we change it to have the same semantics as intersectsBBox?

aka

queryPtSecondary + tolerance >= barMinSecondary && queryPtSecondary - tolerance <= barMaxSecondary

This corresponds with the fact that the queryPt has an invisible rectangle around itself rather than the bbox of the bar having that invisible rectangle